README.md

```diff
@@ -45,91 +61,40 @@ Below is a sample playbook:
-
-
-#### Task : activate.yml
-###### action: activate
-
-The <code>activate</code> task activates the Deep Security Agent by registering it in Trend Micro Deep Security Manager. By default, activation is skipped if the agent is already activated, unless the <code>force_reactivation</code> attribute is set to <code>true</code>.
-
-Key | Type | Description | Default
-----|------|-------------|--------
-dsm_agent_activation_hostname | String | The hostname for the agents to communicate with once they are deployed. For Marketplace and software deployments this is typically the same hostname as 'dsm_agent_download_hostname'. | agents.deepsecurity.trendmicro.com
-dsm_agent_activation_port | Int | The port to use for the agent heartbeat (the regular communication). For Marketplace and software deployments, the default is 4120. | 443
-tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. |
-token/tenant_password | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. In latest Deep Security Manager, "tenant_password" has been replaced with "token". "tenant_password" has been kept for backwards compatibility |
-policy_id | String | The Deep Security ID assigned to the policy and applied to the agents on activation. |
-group_id | String | The Deep Security ID assigned to the computer group and applied to the agents on activation. |
-force_reactivation | Boolean | Force re-activation even though the Deep Security Agent has already been activated. | false
-
-
-#### Task : manage.yml
-###### action:
-
-* check-in-with-manager
-* create-diagnostic-package
-* create-integrity-baseline
-* run-recommendation-scans
-* scan-for-integrity-changes
-* scan-for-malware
-
-The <code>manage</code> task lets you operate the Deep Security Agent without requiring you to log in to the Deep Security Manager. All of the available actions assume that the Deep Security Agent is installed and activated properly. Most of the actions can be triggered without any additional variables. Only <code>set-policy</code> and <code>set-policyid</code> need a corresponding policy name or id.
-
-###### action: set-policy-by-name
-
-Key | Type | Description | Default
-----|------|-------------|--------
-policy_name | String | The name of the Deep Security policy | 
-
-###### action: set-policy-by-id
-
-Key | Type | Description | Default
-----|------|-------------|--------
-policy_id | Int | The ID of the Deep Security policy | 
-
-```yaml
-- hosts: all
-  roles:
-    - role: deep-security.deep-security-agent
-      action: check-in-with-manager | create-diagnostic-package | ...
-      
-- name: "Set policy for webservers"
-  hosts: web
-  roles:
-    - role: deep-security.deep-security-agent
-      action: set-policy-by-name
-      policy_name: MyCompany - Webserver            
-```
+## Actions Definition
```

Comment:
I'll suggest to move the Actions section above the Variable section with a reference link to see the detail.
For users, they'll need to know which action they'd like to use first before setting variables.